feat(curriculum): add css-penguin practice project (#44214)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/01-responsive-web-design/learn-css-transforms-by-building-a-penguin/step-075.md

---
id: 619d1c5fc9f8941a400955da
title: Step 75
challengeType: 0
dashedName: step-75
---

# --description--

> You might have noticed the _heart_ emoji lost its color. This is because some of the character's properties were overridden by the `font-weight` style of `bold`.

Reviewer: ```suggestion
If you look closely, the _heart_ emoji is slightly different. This is because some of the character's properties were overridden by the `font-weight` style of `bold`.
```

This is minor, but I didn't notice the heart lose it's color at all. It looked slightly different if I looked really close (firefox/chrome).  So the wording didn't make sense to me. Perhaps you see something different?

Author: What I expected:
<img width="72" alt="image" src="https://user-images.githubusercontent.com/51722130/144512581-5f8e0582-44c0-4490-81f4-50c88a11eafe.png">

VS
<img width="68" alt="image" src="https://user-images.githubusercontent.com/51722130/144512620-7ffb21bc-acbc-42a2-a394-8c7aee54743e.png">

Author: Firefox handles this nicely, but differently.

Reviewer: What browser is the top? Any extensions messing with something? I saw all purple on chrome and firefox - no outline though (screen shot in review comment).

Author: They are the same browser (Chrome), just one without `.shirt div { font-weight: initial; }` and one with. No extensions, and nothing added that I could see.

Reviewer: Well, I'm fine with it how it is - pretty minor. Feel free to change it up if you want.
